pkg/watch/health.go

```diff
@@ -156,13 +156,15 @@ func updatePods(
 
 		// if a manifest is in reality but not current a podwatch is created
 		// with that manifest and added to newCurrent
-		if missing && man.Manifest.StatusPort != 0 {
+		if missing {
 			sc := StatusChecker{
 				ID:     man.Manifest.Id,
 				Node:   node,
 				Client: client,
 			}
-			if man.Manifest.StatusHTTP {
+			if man.Manifest.StatusPort == 0 {
+				sc.URI = ""
```

Comment:
Probably A Bad Idea ®: make the default URI an endpoint on the _preparer_ that effectively emits the result of `sv stat` for a given pod ID.